x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/service.ts

```diff
@@ -129,11 +129,16 @@ export async function updateOneHostIsolationExceptionItem(
     body: JSON.stringify(entry),
   });
 }
-export async function getHostIsolationExceptionSummary(
-  http: HttpStart
-): Promise<ExceptionListSummarySchema> {
+export async function getHostIsolationExceptionSummary({
+  http,
```

Comment:
could we add the filter as a second param instead of adding it in an object?
Something like:

```
function getHostIsolationExceptionSummary(
  http: HttpStart,
  filter?: string
) {}
```

Reply:
I like the destructured version whenever there is more than one parm, that way don't have to pass the arguments in a specific order. TS may not always complain, esp. if the types are the same for the params.

Reply:
Yeah, that make sense to me but when having related param values. Http seems very different than filter one so is a bit weird having it in the same object but it's just an opinion.

Reply:
Yeah makes sense. I'll just go ahead and make the change 👍

Reply:
done 39e4ba34b1ad8e4f090551ba1f4c9aef6f5b1c83